pynumdiff/kalman_smooth/_kalman_smooth.py

```diff
@@ -278,7 +278,7 @@ def robustdiff(x, dt, order, log_q, log_r, proc_huberM=6, meas_huberM=0):
     proper :math:`\\ell_1` normalization.
 
     Note that :code:`log_q` and :code:`proc_huberM` are coupled, as are :code:`log_r` and :code:`meas_huberM`, via the relation
-    :math:`\\text{Huber}(q^{-1/2}v, M) = q^{-1}\\text{Huber}(v, Mq^{-1/2})`, but these are still independent enough that for
```

Comment:
This was just straight up wrong and caused me some confusion. But it's fixed now, and the math in the paper in in various code around here are modified to reflect a slightly simpler form. It's nice to just modify the M instead of divide inputs and scale the whole loss function.